routingtable/routingtable/rib_interface.go

```diff
@@ -0,0 +1,11 @@
+package routingtable
```

Comment:
this is a copy of the routingtable package in the routingtable package and is considered a duplicate ;)